src/Microsoft.OpenApi/Models/OpenApiDocument.cs

```diff
@@ -524,20 +524,37 @@ public void SetReferenceHostDocument()
         /// <returns>The hash value.</returns>
         public async Task<string> GetHashCodeAsync(CancellationToken cancellationToken = default)
         {
+            byte[]? hash;
+
+#if NET
```

Comment:
```suggestion
#if NET7_OR_GREATER
```
https://learn.microsoft.com/en-us/dotnet/api/system.security.cryptography.sha512.hashdataasync?view=net-9.0